src/i2c.rs

```diff
@@ -148,7 +152,7 @@ where
         I2c {
             i2c,
             pins,
-            timeout: 255,
+            timeout: 1255,
```

Comment:
Any reason for chosing this number?

Reply:
that is just a random number that worked for me since 255 was too short when running at 160Mhz - I change it 2048 since 1255 looks a bit strange